### PR TITLE
Allow the code generator to generate only the low level client

### PR DESCRIPTION
**Pull request by Mpdreamz**
Isolated from: #4336

This allows us to start generating the low level client ahead of time and automate this in the future.

### CODE REVIEW

**Comment by Mpdreamz** [comment]
Please merge https://github.com/elastic/elasticsearch-net/pull/4347 first and use rebase and commit so that the commit from https://github.com/elastic/elasticsearch-net/pull/4347 stays isolated

File: src/ApiGenerator/Configuration/GeneratorLocations.cs [hunk 1]
```diff
@@ -27,15 +27,15 @@ public static string Root
 
 				var directoryInfo = new DirectoryInfo(Directory.GetCurrentDirectory());
 
-				var runningAsDnx =
+				var dotnetRun =
```
Comment (russcam) [inline src/ApiGenerator/Configuration/GeneratorLocations.cs]: better name for it now 😃

File: src/ApiGenerator/Program.cs [hunk 2]
```diff
@@ -40,18 +32,29 @@ private static async Task Main(string[] args)
...
+
+		private static bool Ask(string question, bool defaultAnswer = true)
+		{
+			var answer = "invalid";
+			var defaultResponse = defaultAnswer ? "y" : "n";
+
 			while (answer != "y" && answer != "n" && answer != "")
 			{
-				Console.Write("Generate code from the specification files on disk? [Y/n] (default Y): ");
+				Console.Write($"{question}[y/N] (default {defaultResponse}): ");
```
Comment (russcam) [inline src/ApiGenerator/Program.cs]: nitpick: The default answer in `[y/n]` should be the capitalized one

Comment (Mpdreamz) [reply src/ApiGenerator/Program.cs]: Good call out will take this on board when introducing command line arguments 👍

**Pull request merged**